flake8_functions/checker.py

```diff
@@ -127,7 +127,13 @@ def run(self) -> Generator[Tuple[int, int, str, type], None, None]:
                 func_def_info, self.max_function_length,
             ):
                 yield lineno, col_offset, error_msg, type(self)
-            error_info = self._get_arguments_amount_error(func_def, self.max_parameters_amount)
-            if error_info:
-                full_error_info = *error_info, type(self)
-                yield full_error_info
+
+            error_amount_info = self._get_arguments_amount_error(func_def, self.max_parameters_amount)
+            if error_amount_info:
+                full_error_amounnt_info = *error_amount_info, type(self)
+                yield full_error_amounnt_info
+
+            error_pure_info = check_purity_of_functions(func_def)
+            if error_pure_info:
```

Comment:
This is seems like copy-paste of if above (line 132). It would be nice to refactor this.